walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,45 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     throw new UnsupportedOperationException("TODO: Implement this method.");
```

Comment:
It seems like this line should go away? Alternatively, if the changes in this function aren't really part of this PR, it probably makes sense to remove them.